### PR TITLE
Create staging checklist for mobile apps

### PR DESCRIPTION
**Pull request by onort**

### CODE REVIEW

File: checklists/staging-mobile-app.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Stage A Mobile App
+category: Mobile
+date: "2020-03-11"
+tags: ['mobile', 'react-native']
+description: Follow these steps during staging a mobile application 
+---
+
+- [ ] Create a new branch, ideally named "testflight"
```
Comment (ecempehlivan) [inline checklists/staging-mobile-app.md]:
```suggestion
- [ ] Create a new branch  
Ideally named "testflight"  
```

File: checklists/staging-mobile-app.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Stage A Mobile App
+category: Mobile
+date: "2020-03-11"
+tags: ['mobile', 'react-native']
+description: Follow these steps during staging a mobile application
```
Comment (akemalakcay) [inline checklists/staging-mobile-app.md]:
```suggestion
description: Follow these steps while staging a mobile application.
```

File: checklists/staging-mobile-app.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Some projects may have been configured to do this task using Fastlane. Run desired fastlane job/-s or do it manually if necessary.
+
+- [ ] Make sure you have set up your authorization correctly  
+You'll need a keystore file for android and authorized certificate/-s for iOS. While uploading to AppStore you might also need to enter a SMS verification code.
+
+- [ ] Make sure staging application has the correct configuration  
+API endpoints or environment variables may differ.
+
+- [ ] Build a staging application  
+Built APK file for Android can be found inside projects android folder `app/build/output/apk/release/`, iOS application will be sent to TestFlight.
```
Comment (akemalakcay) [inline checklists/staging-mobile-app.md]:
```suggestion
Built APK file for Android can be found inside project's Android folder `app/build/output/apk/release/`, iOS application will be sent to TestFlight. 
```

File: checklists/staging-mobile-app.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+- [ ] Create a new branch  
+Ideally named "testflight"  
+
+- [ ] Update version and build number  
+Some projects may have been configured to do this task using Fastlane. Run desired fastlane job/-s or do it manually if necessary.
+
+- [ ] Make sure you have set up your authorization correctly  
+You'll need a keystore file for android and authorized certificate/-s for iOS. While uploading to AppStore you might also need to enter a SMS verification code.
```
Comment (akemalakcay) [inline checklists/staging-mobile-app.md]:
```suggestion
You'll need a keystore file for Android and authorized certificate/-s for iOS. While uploading to AppStore you might also need to enter a SMS verification code.
```

**Comment by onort** [comment]
> Do we need to mention uploading dSYSM files to Crashlytics? What do you think? @onort

I think uploading dSYM files should be a part of a production build checklist. @alpcanaydin

**Pull request merged**